Android/PulaJaswanthRam/Intermediate/ToDoApplication/app/src/main/java/com/example/todoapplication/WelcomePage.java

```diff
@@ -0,0 +1,30 @@
+package com.example.todoapplication;
+
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+public class WelcomePage extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_welcome_page);
+        
+
+        ActionBar actionBar = getSupportActionBar();
+        actionBar.hide();
+
+        Handler handler = new Handler();
```

Comment:
The handler is deprecated. Use the recommendation and try using lambda in place of Anonymous new Runnable().